ps3.py

```diff
@@ -0,0 +1,361 @@
+# 6.0001 Problem Set 3
+#
+# The 6.0001 Word Game
+# Created by: Kevin Luu <luuk> and Jenna Wiens <jwiens>
+#
+# Name          : Bogdan Perederei
+# Collaborators : -
+# Time spent    : 4 hours
+
+import math
+import random
+import string
+import re
+
+WILDCARD = '*'
+STOP_GAME_SYMBOL = '!!'
+VOWELS = 'aeiou'
+CONSONANTS = 'bcdfghjklmnpqrstvwxyz'
+HAND_SIZE = 7
+
+SCRABBLE_LETTER_VALUES = {
+    'a': 1, 'b': 3, 'c': 3, 'd': 2, 'e': 1, 'f': 4, 'g': 2, 'h': 4, 'i': 1, 'j': 8, 'k': 5, 'l': 1, 'm': 3, 'n': 1,
+    'o': 1, 'p': 3, 'q': 10, 'r': 1, 's': 1, 't': 1, 'u': 1, 'v': 4, 'w': 4, 'x': 8, 'y': 4, 'z': 10, WILDCARD: 0
```

Comment:
WILDCARD is not a letter, please - remove it from here

Reply:
done